libs/ui/tabs/brain/src/lib/brn-tabs-trigger.directive.ts

```diff
@@ -37,6 +37,10 @@ export class BrnTabsTriggerDirective {
 	@Input()
 	public disabled = false;
 
+	public setTriggerFor(key: string) {
```

Comment:
should this be inputs/setters instead of functions?

Reply:
I also was not happy with that, but I was not able to call the setter directly in the hlm.
Because of that, I introduced that function and was hoping that you maybe have a solution for that. ;)

Reply:
```
this._brn?.triggerFor = 'test'; 
```
does not work. _The left-hand side of an assignment expression may not be an optional property access_
but the following works. 
```
if (this._brn) {
  this._brn.triggerFor = 'test';
}
```
I removed the extra function